### DISCUSSION TITLE
Child menu item is not expandable

### DISCUSSION THREAD

**Discussion by ramonov** [balajidharma/laravel-vue-admin-panel#20] [General]
Dear balaji, I found after creating child menu item from backend its not being expanded while clicking on parent + sign.

thanks

**Comment by balajidharma** [comment]
@ramonov Change the parent menu link to `<nolink>` If you using a child menu. 

![image](https://user-images.githubusercontent.com/6037466/226769235-04e5bf4a-7ff5-4635-a5b9-4b15251ca924.png)
